clients/language-server/visual-studio-code/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I think since we plan to get this into the existing OPA client, we can remove the client all together.

Reply:
Right — I'm suggesting we remove the server parts, not the client. We're not building a server as part of this.

Reply:
I guess what I was trying to get at was that we have an almost identical file in the `client` dir. Perhaps we wouldn't need to have one file coordinating the build of both a client and a server if we're only building the client in JS/TS?

Reply:
Sure, but I think that most of this will end up going into the OPA extension instead right? Since it's a generic language server, I hope that we can extend the OPA ext with generic use of $some_lang_server, including Rego? Or is that not what you were expecting?

Reply:
👍 I'll drop the client from this PR and we can catch up next week!